### PR TITLE
Corrected arch name

### PR DESCRIPTION
**Pull request by ChenSun-YP**
Corrected variable name 'arch'  to the appropriate names.  This ensures downloading the pre-trained model from the correct source. In the original version  calling 
    model = spiking_vgg.spiking_vgg16_bn(pretrained=True, spiking_neuron=neuron.IFNode,norm_layer=BatchNorm2d,
                                      surrogate_function=surrogate.ATan(), detach_reset=True)
would download pretrained model without batchnorm2d.

### CODE REVIEW

**Comment by fangwei123456** [comment]
Thanks for your debug!

**Comment by ChenSun-YP** [comment]
welcome ;)
